PL.ConsoleClient/Program.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Threading.Tasks;
+using BLL.CoreEntities.Entities;
+
+namespace PL.ConsoleClient
+{
+    class Program
+    {
+        static HttpClient client = new HttpClient();
+        private const string itemIdFromDb = "60";
+        static void ShowProduct(Product product)
+        {
+            Console.WriteLine($"Name: {product.ProductName}\tPrice: " +
+                              $"{product.UnitPrice}\tCategory: {product.Category}");
+        }
+        static async Task<Product> GetProductAsync(string path)
```

Comment:
Please use formatters to format your code. (between methods should be one empty line)